fsmonitor.c

```diff
@@ -15,7 +15,8 @@ static void fsmonitor_ewah_callback(size_t pos, void *is)
 {
 	struct index_state *istate = (struct index_state *)is;
 	struct cache_entry *ce = istate->cache[pos];
-
+	
```

Comment:
I can’t see the whole context on my iPad, but I’d consider changing this a bit. 

assert() is a debug-only macro, so it’s wont give production protection.  Suggest “if (pos < …) BUG(…);” instead.  Or maybe just “return” if that’s more appropriate.

Also, maybe split the declaration and assignment of “ce”,  but then again as long as you don’t deref the pointer, that’s more of a nit.